app/konnect-reference-platform/faq.md

```diff
@@ -0,0 +1,224 @@
+repository is the central repository for API specifications and APIOps workflows and execution.
+
+`teams`
+
+The `teams` configuration defines the engineering group's developer teams and their application repositories. Every team defined 
+here will correspond to a Konnect Team resource within the organization. Each service application repository defined here will have its 
+API specifications read and staged into the Platform repository. Simple user management can be accomplished in this section if IdP is 
+not utilized. Providing email addresses for teams will invite users to the Konnect organization.
+
+`organizations`
+
+Each organization defined in the `organizations` section, maps to a Konnect Organization. For each organization, you define 
+authorization configurations and environments. Each environment will result in a set of configured resources with specific naming, 
+metadata, and access control policies.
+
+## What are Environments?
+Environments may be any conceptual grouping of resources the user desires. Typical examples include "dev" or "prod" engineering stages,
+but you may also desire to form environments around business units or products. 
+
+Konnect does not natively support the concept of environments. The orchestrator manages environments synthetically by 
+prefixing resource names, applying labels, and setting different access control policies based on well known environment types. 
+The orchestrator currently supports 2 environment types, DEV and PROD. Various resource configuration decisions are encoded into the Environment type, 
+for example RBAC and portal settings. Environments also require a region configuration which must map to a Konnect supported geographic region.
+
+## What specific Konnect resource are managed by the Konnect Orchestrator?
+The Konnect Orchestrator does not manage a straight mapping from the input declarative configuration into Konnect resources. 
+Instead, the input configuration is generally an expression of the engineering organization design, and the orchestrator 
+maps that to opinionated configurations for the Konnect resources it manages. The following shows the general configuration
+of Konnect resources managed by the orchestrator:
+
+* For each configured organization in the `organizations` configuration
+    * For every team in the `teams` configuration
+        * A Konnect Team is created 
+            * For every User in the `users` configuration, a Konnect User is created 
+            * Each User is added to the Konnect Team
+        * For every environment in the `environments` configuration
+            * A Konnect Control Plane is created
+            * A Konnect Developer Portal is created
+            * For every service in the `services` configuration
+                * A Konnect API is created
+                * The service specification is added to the API
+
```

Comment:
I don't really understand this. Is this basically a for loop? It's hard to read as-is. I wonder if there's a way to display this as tree structure diagram.